CMakeLists.txt

```diff
@@ -30,11 +30,11 @@ endif (NOT CMAKE_INSTALL_PREFIX)
 if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     message(STATUS "==============OSX Environment================")
     set(MACOS TRUE)
-    set(CMAKE_CXX_COMPILER "/usr/local/opt/llvm/bin/clang++")
-    set(CMAKE_CXX_FLAGS "-I/usr/local/opt/llvm/include -I/usr/local/include -I/usr/local/opt/openblas/include -I/usr/local/opt/libomp/include")
+    # Adjust accordingly if libomp is not install here.
+    set(CMAKE_CXX_FLAGS "-I/opt/homebrew/Cellar/libomp/13.0.0/include")
```

Comment:
how to ensure this path work on other mac laptops?

Reply:
Forgot to change back.

Fixed.